cocos/2d/CCActionInterval.cpp

```diff
@@ -327,11 +328,10 @@ void Sequence::startWithTarget(Node *target)
         log("Sequence::startWithTarget error: _actions[0] or _actions[1] is nullptr!");
         return;
     }
-    if (_duration <= FLT_EPSILON)
-        return;
+    if (_duration > FLT_EPSILON)
+        _split = _actions[0]->getDuration() / _duration;
```

Comment:
Yep, @WenhaiLin describe it to me in other way. If _duration == FLT_EPSILON, then we should not do `_split = _actions[0]->getDuration() / _duration` since it will be an error.

@stevetranby 
Is there any problem?